src/games/strategy/engine/framework/ServerGame.java

```diff
@@ -360,73 +359,45 @@ public void stopGame() {
   }
 
   private void autoSave() {
-    FileOutputStream out = null;
-    try {
+    final File f1 = new File(SaveGameFileChooser.DEFAULT_DIRECTORY, SaveGameFileChooser.getAutoSaveFileName());
+    final File f2 = new File(SaveGameFileChooser.DEFAULT_DIRECTORY, SaveGameFileChooser.getAutoSave2FileName());
+    final File f;
+    if (f1.lastModified() > f2.lastModified()) {
+      f = f2;
+    } else {
+      f = f1;
+    }
+
+    try (FileOutputStream out = new FileOutputStream(f)) {
       SaveGameFileChooser.ensureDefaultDirExists();
```

Comment:
Is it possible that "ensureDefaultDirExists needs to come first, before the File() constructor statements?
I think so, at least at first glance...